utils/src/ooinstall/openshift_ansible.py

```diff
@@ -153,17 +156,17 @@ def write_inventory_vars(base_inventory, multiple_masters, lb):
 def write_proxy_settings(base_inventory):
     try:
         base_inventory.write("openshift_http_proxy={}\n".format(
-                                                            CFG.settings['openshift_http_proxy']))
+            CFG.settings['openshift_http_proxy']))
```

Comment:
tl;dr: I think it's fine just the way it is.

@detiber any of the first three examples in the PEP8 spec section for indentation should be preferred:

https://www.python.org/dev/peps/pep-0008/#indentation

```python
# Aligned with opening delimiter.
foo = long_function_name(var_one, var_two,
                         var_three, var_four)

# More indentation included to distinguish this from the rest.
def long_function_name(
        var_one, var_two, var_three,
        var_four):
    print(var_one)

# Hanging indents should add a level.
foo = long_function_name(
    var_one, var_two,
    var_three, var_four)
```

I wrote up this:
```python
    try:
        base_inventory.write("openshift_http_proxy={}\n".format(
            CFG.settings['openshift_http_proxy']))
```

and you're suggesting this:

```python
    try:
         base_inventory.write("openshift_http_proxy={}\n"
                              .format(CFG.settings['openshift_http_proxy']))
```

In my approach, *technically*, I think there shouldn't be any arguments on the first line. Quoth the PEP:

> When using a hanging indent the following should be considered; there should be no arguments on the first line

However, since both of these pass the PEP8 test I favor my original suggestion because it is more compact visually and the alternative looks more confusing:

```python
    try:
        base_inventory.write(
            "openshift_http_proxy={}\n".format(
                CFG.settings['openshift_http_proxy']))
```

And as always, not all rules are hard and fast:

> know when to be inconsistent -- sometimes style guide recommendations just aren't applicable. When in doubt, use your best judgment ... Some other good reasons to ignore a particular guideline ... When applying the guideline would make the code less readable, even for someone who is used to reading code that follows this PEP.

https://www.python.org/dev/peps/pep-0008/#a-foolish-consistency-is-the-hobgoblin-of-little-minds